README.md

```diff
@@ -9,7 +9,31 @@ Run the installation script with the optional hooks
 $ ./install --hooks
 ```
 
-To start the server locally, run `npm start`
+Make sure you add `stopuottawa.local` to your `/etc/hosts` file so it routes you correctly.
```

Comment:
Why this domain?? I thought we were using `upetition.ca`?? Also if this is for internal routing these routes should be linked through docker not externally

Reply:
This is for development only and mostly because I forgot what we had set it to, I'll change the routes

Reply:
Ok but if we don't add it and just navigate to localhost it should still work right?? So it's not required as such??

Reply:
I didn't add 'localhost' as a server name in nginx so it *probably* wouldn't but I haven't tested that